src/wiki/getting-started/controller-support.md

```diff
@@ -22,9 +22,15 @@ Finally, if you don't have one already, you must create an **instance** with the
 
 Before you continue, ensure that the correct version of the **Fabric API** mod on **Fabric** or the **Quilted Fabric API** mod on **Quilt** for your instance is installed.
 
+### <img src="https://raw.githubusercontent.com/MrCrayfish/Controllable/6caef1a4ac113e5c6ac1d1abde0f0cabc3e6ad97/src/main/resources/controllable_icon.png" height="20">  Controllable (Fabric) (for Minecraft Versions 1.18 or newer)
+
+For Minecraft versions 1.18 or above, we recommend [**Controllable (Fabric)**](https://www.curseforge.com/minecraft/mc-mods/controllable-fabric).
```

Comment:
I think we should still recommend midnightcontrols as its on modrinth and doesn't have any dependency and just list controllable for fabric as an alternative

Reply:
That sounds like a great idea. Though you should definitely change "Controllable" to "Controllable (Forge)" to reflect its name change on the page.